cmake/configs/posix_rpi_common.cmake

```diff
@@ -74,6 +74,7 @@ set(config_module_list
 	#
 	# PX4 drivers
 	#
+	drivers/linux_sbus
```

Comment:
Can you try to please add this to the other posix `.cmake` and test if it builds?